File: server/domain/device_item.py
class Device_item:
    column_id = ''
    device_id = ''
    device_item_id = ''
    name = ''
    unit = ''

    def get_Data(self):
        return {
            "id": self.column_id,
            "deviceID": self.device_id,
            "deviceItemID": self.device_item_id,
            "name": self.name,
            "unit": self.unit
        }

File: client/src/domain/DeviceItem.js
export default class DeviceItem {
    constructor(column_id = '', device_id = '', device_item_id = '', name = '', unit = '') {
        this.column_id = column_id
        this.device_id = device_id
        this.device_item_id = device_item_id
        this.name = name
        this.unit = unit
        this.isVisible = true
    }
}